AEPTarget/Sources/TargetConstants.swift

```diff
@@ -195,14 +199,23 @@ enum TargetConstants {
         static let IS_LOCATION_DISPLAYED = "islocationdisplayed"
         static let IS_LOCATION_CLICKED = "islocationclicked"
         static let MBOX_PARAMETERS = "parameters"
-        static let ORDER_PARAMETERS = "orderparameters"
-        static let PRODUCT_PARAMETERS = "productparameters"
-        static let PROFILE_PARAMETERS = "profileparameters"
+        static let ORDER_PARAMETERS = "order"
+        static let PRODUCT_PARAMETERS = "product"
+        static let PROFILE_PARAMETERS = "profileParameters"
```

Comment:
this concerns me - is this fixing a bug, or are those constants being used for something else?

Reply:
Those constants were unused!